lib/propmangle.js

```diff
@@ -150,6 +150,11 @@ function mangle_properties(ast, options) {
             if (!(keep_quoted && node.quote))
                 node.key = mangle(node.key);
         }
+        else if (node instanceof AST_ConciseMethod && node.name && node.key.name) {
```

Comment:
This PR was adapting `AST_ConciseMethod` code that was either not reachable or not working.